### DISCUSSION TITLE
Print the timestamp

### DISCUSSION THREAD

**Discussion by amitmthorat** [Banno/Gordon#129] [Q&A]
Hi @joshschriever, I wanted to know if there is any option/setting so that I can print the time taken by each test or time taken to run the complete test suite. I'm using v1.8.7.

**Comment by joshschriever** [comment]
No, not currently. We could move this discussion to the "ideas" category - I'm sure it wouldn't be too hard to implement. Of course, releasing it in a new version won't help if you're stuck on an old version. Can I ask why you're using v1.8.7?

**Reply by amitmthorat** [reply]
Hi @joshschriever, old version is due to older java version soon I'll update the java version. Another query is, I want skip few tests those marked with some annotation. I used **testFilter.set("com.vidyamitra.support.test.espresso.annotations.MslContractTest, com.vidyamitra.support.test.espresso.annotations.EndToEndTest")** but it's not filtering these tests.
Please suggest.
Thanks you!

**Reply by joshschriever** [reply]
The test filter is a _positive_ filter, so that's telling Gordon which tests to _run_, not which tests to skip. For skipping tests with certain annotations you can use Android's `notAnnotation` instrumentation runner argument - https://github.com/Banno/Gordon/tree/1.8.7#compatibility-with-android-extension.
